src/spring-cloud/azext_spring_cloud/_params.py

```diff
@@ -315,3 +317,22 @@ def prepare_logs_argument(c):
                   'spring-cloud application-configuration-service git repo remove']:
         with self.argument_context(scope) as c:
             c.argument('name', help="Required unique name to label each item of git configs.")
+
+    for scope in ['spring-cloud build-service buildpacks-binding create',
+                  'spring-cloud build-service buildpacks-binding set']:
+        with self.argument_context(scope) as c:
+            c.argument('type', arg_type=get_enum_type(BuildpacksBindingType), help='Required type for buildpacks binding.')
+            c.argument('properties',
+                       help='Non-sensitive properties for launchProperties. Sample "key1=val1 key2=val2"',
+                       validator=validate_buildpacks_binding_properties)
+            c.argument('secrets',
+                       help='Sensitive properties for launchProperties. Once put,it will be encrypted and never return to user. Sample "sec1=val1 sec2=val2"',
+                       validator=validate_buildpacks_binding_secrets)
+
+    for scope in ['spring-cloud build-service buildpacks-binding create',
+                  'spring-cloud build-service buildpacks-binding set',
+                  'spring-cloud build-service buildpacks-binding show',
+                  'spring-cloud build-service buildpacks-binding delete']:
+        with self.argument_context(scope) as c:
+            c.argument('service', service_name_type, validator=only_support_enterprise)
+            c.argument('name', help='Reqired name for buildpacks binding.', validator=validate_buildpacks_binding_name)
```

Comment:
Remove “Reqired”, the name is already required

Reply:
Sure